ChatSecure/Classes/View Controllers/OTRRoomOccupantsViewController.swift

```diff
@@ -10,89 +10,221 @@ import Foundation
 import UIKit
 import PureLayout
 
-public class OTRRoomOccupantsViewController: UIViewController {
+open class OTRRoomOccupantsViewController: UIViewController {
     
-    let tableView: UITableView
-    let viewHandler: OTRYapViewHandler
-    let roomKey: String
+    @IBOutlet open weak var tableView:UITableView!
+    @IBOutlet weak var largeAvatarView:UIImageView!
+    
+    open var viewHandler:OTRYapViewHandler?
+    open var roomKey:String?
+    open var room:OTRXMPPRoom?
+    open var headerRows:[String] = []
+    open var footerRows:[String] = []
```

Comment:
Yes, subclasses should be able to access headers and footers to add/remove/config at will.